src/main/java/bluey.java

```diff
@@ -0,0 +1,59 @@
+import java.util.Scanner;
+
+public class bluey {
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm bluey!");
+        System.out.println("What can I do for you?\n");
+
+        TaskControl taskControl = new TaskControl();
+        Scanner in = new Scanner(System.in);
+        while (true) {
+                String userResponse = in.nextLine().trim();
+                if (userResponse.equals("exit") || userResponse.equals("bye")) {
+                    break;
+                }
+                handleResponse(userResponse, taskControl);
+//            } catch (InvalidCommandException e) {
+//                System.out.println(e.getMessage());
+////            } catch (ArrayIndexOutOfBoundsException e) {
+//
+//            }
+        }
+        System.out.println("Goodbye! See you soon :)");
+    }
+
+    public static void handleResponse(String userResponse, TaskControl taskControl) {
+        String[] words = userResponse.split("\\s+");
+        switch (words[0]) {
+        case "list":
+            taskControl.printList();
+            break;
+        case "mark":
```

Comment:
Note down your fallthrough cases, this can be a very dangerous way of coding so be very careful about it

https://se-education.org/guides/conventions/java/basic.html#layout
Check the switch statement format.